File: ExtApp/ExtApp.Model/Workflow/Workflow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using NHibernate.Mapping.Attributes;

namespace ExtApp.Model
{
    /// <summary>
    /// 工作流
    /// </summary>
    [Class(Table = "Workflow")]
    public class Workflow
    {
        /// <summary>
        /// ID
        /// </summary>
        [Id(Name = "ID")]
        [Generator(Class = "identity")]
        public virtual int ID { get; set; }

        /// <summary>
        /// 名称
        /// </summary>
        [Property]
        public virtual string Name { get; set; }

        /// <summary>
        /// 版本
        /// </summary>
        [Property]
        public virtual int Version { get; set; }

        /// <summary>
        /// 工作流数据
        /// </summary>
        [Property(Length = 10000)]
        public virtual string Data { get; set; }

        /// <summary>
        /// 添加人
        /// </summary>
        [ManyToOne(ClassType = typeof(User), Column = "AddUserID", Lazy = Laziness.False)]
        public virtual User AddUser { get; set; }

        /// <summary>
        /// 添加时间
        /// </summary>
        [Property]
        public virtual DateTime AddTime { get; set; }

        /// <summary>
        /// 备注
        /// </summary>
        [Property]
        public virtual string Comment { get; set; }
    }
}

File: ExtApp/ExtApp.Web/app/model/workflow/Workflow.js

Ext.define('App.model.workflow.Workflow', {
    extend: 'Ext.data.Model',

    fields: [
        'ID',
        'Name',
        'Version',
        'Data',
        {
            name: 'AddUserID',
            mapping: function (value) {
                if (value.AddUser == null) {
                    return 0;
                }
                return value.AddUser.ID;
            }
        },
        {
            name: 'AddUserName',
            mapping: function (value) {
                if (value.AddUser == null) {
                    return '';
                }
                return value.AddUser.Name;
            }
        },
        'AddTime',
        'Comment'
    ]
});